routes/api/chromes.js

```diff
@@ -0,0 +1,50 @@
+const router = require('express').Router();
+
+const { Chrome }= require('../../databases/db');
+
+//endpoint para listar cromos
+router.get('/', async (req,res)=>{
+    //paginacion
+    const {page = 0, size = 10} = req.query;
+
+    let options = {
+        limit: +size,
+        offset: (+page) * (+size)
+    };
+    const users = await Chrome.findAll(options);
+    res.status(200).json({message: 'Lista de usuarios', users });
```

Comment:
```suggestion
    res.status(200).json({message: 'Lista de usuarios', chromes });
```
Devuelve los cromos, no los usuarios. Cambio sugerido para mejorar la claridad